### PR TITLE
chore(build): adjust settings in package.json

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,29 @@
 {
-  "name": "angular-ng-bootstrap",
-  "version": "0.1.0",
-  "description": "ng-bootstrap core components for angular2",
-  "author": "https://github.com/ng-bootstrap/core/graphs/contributors",
-  "main": "dist/cjs/core.js",
+  "name": "ng-bootstrap",
+  "version": "1.0.0-alpa.0",
```
Comment (Foxandxss) [inline package.json]: alpa-cino

Comment (pkozlowski-opensource) [inline package.json]: Good catch, thnx!

File: package.json [hunk 2]
```diff
@@ -1,27 +1,33 @@
 {
-  "name": "angular-ng-bootstrap",
-  "version": "0.1.0",
-  "description": "ng-bootstrap core components for angular2",
-  "author": "https://github.com/ng-bootstrap/core/graphs/contributors",
-  "main": "dist/cjs/core.js",
+  "name": "ng-bootstrap",
+  "version": "1.0.0-alpha.0",
+  "description": "Angular 2 powered Bootstrap",
+  "keywords": "Angular 2 Angular2 Bootstrap",
```
Comment (icfantv) [inline package.json]: won't the "2" show up as a keyword by itself?

**Comment by icfantv** [comment]
Just the issue w/ the keywords and LTGM.

**Pull request closed without merging**